src/lib/rule2A.ts

```diff
@@ -7,11 +7,20 @@ import {Context, getDefaultContext} from './context';
  * @return - whether or not the node is considered hidden
  */
 // #SPEC_ASSUMPTION (A.2) : definition of 'hidden'
-function isHidden(node: Node): boolean {
+function isHidden(node: Node, context: Context): boolean {
   if (!(node instanceof HTMLElement)) {
     return false;
   }
 
+  // #SPEC_ASSUMPTION (A.3) : options shouldn't be hidden
+  if (
+    node instanceof HTMLOptionElement &&
+    node.closest('select') !== null &&
+    context.inherited.partOfName
```

Comment:
is this actually needed ? 

what happens if you make options always not hidden, like in firefox ?

something like:

```ts
if(node instanceof HTMLOptionElement) {
  const parentSelect = node.closest('select');
  if(parentSelect) {
    return isHidden(parentSelect);
  }
}
```

Reply:
Interesting idea yeah, so what do you think should be the acc name for div#foo here:
```
<select id="bar" hidden>
  <option selected>Hello</option>
</select>
<div id="foo" aria-labelledby="bar"></div>
```
In this case the select is the direct label for div#foo, so even though it's hidden the acc name here should probably be 'Hello' I think? In which case we would need to consider whether or not the `<option>` is part of a name rather than just the visibility of its parent `<select>`.